spec/enrichments/tsdoc_v1/storage.json

```diff
@@ -13,7 +13,7 @@
       "flags": {
         "isExported": true
       },
-      "originalName": "/home/runner/work/storage-js/storage-js/src/StorageClient.ts",
+      "originalName": "/Users/copple/Projects/Supabase/storage-js/src/StorageClient.ts",
```

Comment:
Something doesn't look right here, there are references to copple's user directory.

Reply:
OH WOW!

Reply:
I had to run these manually in the repos when we moved to v2, so this is fine